MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -48,9 +48,18 @@ open class BaseInstructionsBannerView: UIControl {
     @IBInspectable
     public var swipeable: Bool = false
     
+    @IBInspectable
+    public var showStepIndicator: Bool = true {
```

Comment:
Could you please add a line in the changelog for this addition?

Reply:
sorry forgot will update!